README.md

```diff
@@ -1 +1,20 @@
-# 100-days-of-productivity
\ No newline at end of file
+# 100-days-of-productivity
+
+**Main target:**
+
+- I will be productive for at least 2 hours every day for the next 100 days.
+
+I was inspired by [Alexander Kallaway](https://github.com/Kallaway)'s #100DaysOfCode challenge.
+
+More details about the challenge can be found here:
+
+- [100daysofcode.com](http://100daysofcode.com/)
+- [github.com/Kallaway/100-days-of-code](https://github.com/Kallaway/100-days-of-code)
+
```

Comment:
should probably switch these two so idea can flow a lot smoother.

could also include more general info about our challenge

```suggestion
# 100-days-of-productivity

I was inspired by [Alexander Kallaway](https://github.com/Kallaway)'s #100DaysOfCode challenge.

More details about the challenge can be found here:

- [100daysofcode.com](http://100daysofcode.com/)
- [github.com/Kallaway/100-days-of-code](https://github.com/Kallaway/100-days-of-code)

**Main target:**

- I will be productive for at least 2 hours every day for the next 100 days.
```